src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js

```diff
@@ -307,8 +307,12 @@ function PipelineController(
    * @param projectId the project id of the sample to remove
    * @param sampleId the sample if to remove
    */
-  vm.removeSample = function(projectId, sampleId) {
-    CartService.removeSample(projectId, sampleId).then(function() {
+  vm.removeSample = function (projectId, sampleId) {
```

Comment:
So I'm assuming this is just changing to the cart functions in `resources/js/apis/cart/cart.js`?

Reply:
Yes that is all that is happening here.